src/renderer/screens/config/ConfigScreen.scss

```diff
@@ -199,6 +195,13 @@
       border-bottom: 1px solid;
       border-bottom-color: rgba(0, 0, 0, 0.2);
 
+      button {
```

Comment:
Just special styling of `button`s that appear in `.ValidationError`s. There is only one use case so far.